HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -23,7 +31,17 @@ public async Task<IActionResult> SaveRepair([FromBody] RepairRequest repairReque
             try
             {
                 var result = await saveRepairRequestUseCase.Execute(repairRequest);
-
+                switch (repairRequest?.ContactDetails?.Type)
+                {
+                    case "email":
+                        await sendAppointmentConfirmationEmailUseCase.Execute(repairRequest.ContactDetails.Value, result,
+                            repairRequest.Time.Display);
+                        break;
+                    case "sms":
+                        await sendAppointmentConfirmationSmsUseCase.Execute(repairRequest.ContactDetails.Value, result,
+                            repairRequest.Time.Display);
+                        break;
```

Comment:
Please add a default case which would throw an exception or log when an unknown contact type is encountered.
```suggestion
                        break;
                    default:
                        throw new NotImplementedException($"Unable to send confirmation via unknown communication type: '{repairRequest?.ContactDetails?.Type}'")
                        break;
```